packages/runtime/runtime-utils/src/dataStoreHelpers.ts

```diff
@@ -84,14 +84,14 @@ export async function requestFluidObject<T = IFluidObject>(
     return response.value as T;
 }
 
-export const create404Response = (request: IRequest) => createResponseError(404, "not found", request);
+export const create404Response = (request?: IRequest) => createResponseError(404, "not found", request);
```

Comment:
See my other comment on debuggability. I think having a request as a required param is good, I'd rather create fake one in your case.